### DISCUSSION TITLE
external link in footer

### DISCUSSION THREAD

**Discussion by AlessandroMcNava** [globaleaks/globaleaks-whistleblowing-software#3445] [Q&A]
I would link to put an external link in the footer, with older version it was very easy using HTML and the tag <a href=""> but now it's not possible. 
I need to link a document, it's very important.

**Comment by evilaliv3** [comment]
@AlessandroMcNava: in the footer like in many other interfaces of the software you could use markdown.

https://github.com/showdownjs/showdown/wiki/Showdown%27s-Markdown-syntax

This will enable you to add links and perform some text formatting.

**Comment by tiagolardacrianca** [comment]
How could i, after uploading some PDFs into the VM where i have globaleaks, using them as links, following the markdown? for instance..

i'm not using docker, so i uploaded Manual.pdf into /var/globaleaks. with markdown in the privacy panel i got [Download of the Manual](../Manual.pdf) .. 

Of course that doesn't work. but how should i reffer to the file in the markdown? where should i put it in the server?

Thank you in advance.

**Comment by tiagolardacrianca** [comment]
With the help of the community in Slack, the solution works: 

> "You can try going through Settings > Files > Files. When uploaded the Download button will show the file url". 

Thanks Ludwig and Giovanni Pellerano (evilaliv3) on Slack. Case closed for me.
